task_1/index.html

```diff
@@ -1,47 +1,140 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-
-<body>   
+<body> 
     <div class="wrapper">
-        <div class="header"></div>
+        <div class="header">
+            <div class="coffee-logo">
+                <div><img class="coffee-image-logo" src="./photo/coffee logo.png" alt="coffee logo"></div>
```

Comment:
use kabab case for image name, ids and class
```suggestion
                <div><img class="coffee-image-logo" src="./photo/coffee-logo.png" alt="coffee logo"></div>
```